modules/restore-determine-state-etcd-member.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-unhealthy-etcd-member.adoc
+
+[id="restore-determine-state-etcd-member_{context}"]
+= Determining the state of the unhealthy etcd member
+
+The steps to replace an unhealthy etcd member depend on which of the following states your etcd member is in:
+
+* The node is not running.
```

Comment:
When users run `oc get nodes`, they will see the conditions in `Ready/NotReady` status, when they run `oc get machine`, they will see `Running/Failed` etc amongst other things hence for the docs I suggest:

if the use of the term Node, the corresponding metric is Ready/NotReady
if the use of the term machine, the corresponding metric is Running/Not Running

when users run oc get node, they would see Ready states

Reply:
Updated terminology across the doc.